test/e2e/lib/gutenberg/gutenberg-editor-component.js

```diff
@@ -391,7 +391,7 @@ export default class GutenbergEditorComponent extends AsyncBaseContainer {
 	 * You can just import the class of the block(s) you want to add and pass it to this function, which
 	 * also means we don't need to couple the block class with this one.
 	 *
-	 * @param { typeof GutenbergBlockComponent } blockClass A block class
+	 * @param { Function } blockClass A block class
```

Comment:
This is one of the types I considered, being that it's the constructor for the class, which is a `Function`. However, we do miss the actual type narrowing here as this suggests it could be any kind of class. 

I've been searching around and didn't find a definitive answer for that, but got some hints here: https://github.com/jsdoc/jsdoc/issues/1349, and that's why I chose to use  `typeof GutenbergBlockComponent`. 

I'm fine leaving this as `Function`, though, it'd be interesting to dive a bit deeper into it later if we have time.